dev/com.ibm.ws.security.acme/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -70,6 +70,8 @@
    <AD id="trustStore"           name="%trustStore"         description="%trustStore.desc"         required="false" type="String" />
    <AD id="trustStorePassword"   name="%trustStorePassword" description="%trustStorePassword.desc" required="false" type="String" ibm:type="password" />
    <AD id="trustStoreType"       name="%trustStoreType"     description="%trustStoreType.desc"     required="false" type="String" />
+   <AD id="httpConnectTimeout"   name="internal" description="internal use only" type="String"  required="false" ibm:type="duration" default="10s" />
```

Comment:
I am wondering if both of these should be higher? 20 or 30s?

Reply:
Set to 30 seconds and made related changes.